lib/bloc/account/account_bloc.dart

```diff
@@ -177,9 +171,22 @@ class AccountBloc {
         });
     }
 
-    _listenUserChanges(Stream<BreezUserModel> userProfileStream, BreezBridge breezLib){
-      userProfileStream.listen((user) { 
+    _listenUserChanges(Stream<BreezUserModel> userProfileStream, BreezBridge breezLib, Device device){
+      userProfileStream.listen((user) {
         _currentUser = user;
+
+        if (user.registered && !_startedLightning) {
+          breezLib.bootstrap().then((done) {
+            _startedLightning = true;
```

Comment:
I am afraid of race conditions here when two subsequent user events are emitted fast. In that case we might end up trying to call the start lightning flow twice. Maybe just move the line: "_startedLightning = true;" right after the "if" ?

Reply:
Good catch, fixed.